olm_deploy/scripts/catalog-build.sh

```diff
@@ -1,8 +1,20 @@
 #!/bin/sh
 set -eou pipefail
 
+IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRY=${IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRY:-$LOCAL_IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRY}
 echo "Building operator registry image ${IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRY}"
-docker build -f olm_deploy/operatorregistry/Dockerfile -t ${IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRY} .
+buildah bud --tls-verify=false -f olm_deploy/operatorregistry/Dockerfile -t ${IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRY} .
 
-echo "Pushing operator registry image ${IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRY}"
-docker push ${IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRY}
+if [ -n ${LOCAL_IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRY} ] ; then
+    oc -n openshift-image-registry port-forward service/image-registry 5000:5000 > /dev/null 2>&1 &
```

Comment:
i'd like to encourage you guys to move away from this pattern of port forwarding from your local machine into the cluster registry for pushing your images and simplify your lives.

Just expose the registry on your cluster via a public route(I can give you the oc command to create a public route for the registry) and push to it directly, or push to quay.io, or push to docker.io.

does this port forward pattern provide some benefit?

Reply:
i guess  the benefit is you don't have to set the value for each person who has their own cluster hostname or quay/docker repository that they need to push to.

Reply:
If there is a public route pattern for our a devcluster's image registry, I will prefer it over quay.io/docker.io due to download times depending on the cluster's region. For example the operator-registry-image is over 800MB big.

Reply:
> If there is a public route pattern for our a devcluster's image registry, I will prefer it over quay.io/docker.io due to download times depending on the cluster's region. For example the operator-registry-image is over 800MB big.

I don't see where your ask has relevance.  This PR establishes pushing images to the clusters internal registry through port-forward and the alternate idea is to expose the internal registry as a route.  Where does quay or docker come into this usecase?

Reply:
i assume @periklis was replying to my suggestion that you guys just push your dev images to quay/docker.io instead of using the cluster registry.

Reply:
Exactly @bparees

Reply:
+1 on keeping the process as fast as possible. I like to test my work often as I make changes.